golang/cosmos/go.mod

```diff
@@ -52,6 +52,7 @@ require (
 	github.com/cosmos/go-bip39 v1.0.0 // indirect
 	github.com/cosmos/gorocksdb v1.2.0 // indirect
 	github.com/cosmos/iavl v0.19.5 // indirect
+	github.com/cosmos/ibc-go v1.5.0
```

Comment:
Why this old ibc-go?  `github.com/cosmos/ibc-go/v4` is already in the `go.mod`.

Reply:
Sorry bad imports fixed.